.github/workflows/main.yml

```diff
@@ -71,6 +71,23 @@ jobs:
       run: >-
         conda build --output-folder=conda/pkg  conda/recipe &&
         conda install tvm -c ./conda/pkg
+    - name: Build iOS RPC@MacOS
+      if: startsWith(matrix.os, 'macOS')
+      run: |
+        IOS_VERSION = "14.0"
```

Comment:
As I know bash syntax requires to avoid usage of space char near "=". Should be changed to  
```
IOS_VERSION="14.0"
```

Reply:
yes, sorry